run_tests.sh

```diff
@@ -9,6 +9,9 @@ if [ "$1" == "distrib" ] ; then
   cd ../../
 fi
 
+eval `opam config env`
+bap-server &
```

Comment:
no, it can't be run without `bap-server` executable, visible in the path. And yes, I think, that it would be a good idea to spawn `bap-server` from qira directly from python code. I'm just not sure about the proper place.